packages/office-addin-sso/src/commands.ts

```diff
@@ -31,6 +31,16 @@ export async function configureSSO(manifestPath: string) {
     }
   }
 
+  // If no expiration date was provided, or if we fail to parse the number, we default to 60 days
```

Comment:
Looks like if we fail to parse a number we bail instead of defaulting to 60.  I think that makes sense, but the comment doesn't match.

Reply:
Good catch, thank you. I will update this comment